accord-core/src/test/java/accord/impl/list/ListRequest.java

```diff
@@ -110,38 +110,38 @@ public void accept(Result success, Throwable fail)
             // TODO (desired, testing): error handling
             if (success != null)
             {
-                node.reply(client, replyContext, (ListResult) success);
+                node.reply(client, replyContext, (ListResult) success, null);
```

Comment:
```suggestion
                node.reply(client, replyContext, (ListResult) success, fail);
```

didn't check `fail` yet